assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -53,276 +62,516 @@ export const PRE_PUBLISH_MESSAGE_TYPES = {
-        ),
-        `${COVER_DIMENSION_WIDTH_PX}x${COVER_DIMENSION_HEIGHT_PX}px`,
-        `${ASPECT_RATIO_LEFT}:${ASPECT_RATIO_RIGHT}`
+      MAIN_TEXT: __(
+        'Choose a bigger image for your story cover',
+        'web-stories'
+      ),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {sprintf(
+              /* translators: %s: minimum cover dimension width X minimum cover dimension height.  */
+              __("Use an image that's at least %s", 'web-stories'),
+              `${COVER_DIMENSION_WIDTH_PX}x${COVER_DIMENSION_HEIGHT_PX}px`
+            )}
+          </li>
+          <li>
+            {sprintf(
+              /* translators: %s: cover dimensions aspect ratio.  */
+              __('Maintain a %s aspect ratio', 'web-stories'),
+              `${ASPECT_RATIO_LEFT}:${ASPECT_RATIO_RIGHT}`
+            )}
+          </li>
+        </DashedList>
       ),
     },
     COVER_WRONG_ASPECT_RATIO: {
-      MAIN_TEXT: __('Story cover image wrong aspect ratio', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: %s: cover dimensions aspect ratio. */
-        __('Should maintain a %s aspect ratio.', 'web-stories'),
-        `${ASPECT_RATIO_LEFT}:${ASPECT_RATIO_RIGHT}`
+      MAIN_TEXT: __(
+        'Choose an image with the correct aspect ratio',
+        'web-stories'
+      ),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {sprintf(
+              /* translators: %s: minimum cover dimension width X minimum cover dimension height.  */
+              __("Use an image that's at least %s", 'web-stories'),
+              `${COVER_DIMENSION_WIDTH_PX}x${COVER_DIMENSION_HEIGHT_PX}px`
+            )}
+          </li>
+          <li>
+            {sprintf(
+              /* translators: %s: cover dimensions aspect ratio.  */
+              __('Maintain a %s aspect ratio', 'web-stories'),
+              `${ASPECT_RATIO_LEFT}:${ASPECT_RATIO_RIGHT}`
+            )}
+          </li>
+        </DashedList>
       ),
     },
     LOGO_TOO_SMALL: {
-      MAIN_TEXT: __('Publisher logo is too small', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: 1: minimum publisher logo dimensions. 2: publisher logo dimension ratio. */
-        __(
-          'Should be at least %1$s and maintain a %2$s aspect ratio.',
-          'web-stories'
-        ),
-        `${PUBLISHER_LOGO_DIMENSION}x${PUBLISHER_LOGO_DIMENSION}px`,
-        `${PUBLISHER_LOGO_RATIO}x${PUBLISHER_LOGO_RATIO}px`
+      MAIN_TEXT: __('Upload a bigger logo', 'web-stories'),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {sprintf(
+              /* translators: %s: minimum publisher logo dimensions. */
+              __("Use an image that's at least %s", 'web-stories'),
+              `${PUBLISHER_LOGO_DIMENSION}x${PUBLISHER_LOGO_DIMENSION}px`
+            )}
+          </li>
+          <li>
+            {sprintf(
+              /* translators: %s: publisher logo dimension ratio. */
```

Comment:
To give some context to my suggestions, merely as an FYI:

When a string is presented to a translator in the UI they're going to see the string as well as any additional translator comment added to it.

It therefore makes sense to use the same translator comment for all instances of the same string, otherwise you'll see the string with three different translator comments added to it.

Reply:
Ah makes sense. Thank you!